src/relayclient/ContractInteractor.ts

```diff
@@ -399,8 +400,39 @@ export default class ContractInteractor {
     return await this.web3.eth.estimateGas(gsnTransactionDetails)
   }
 
+  // equivalent to `eval("blob"+path)` - but without evil eval
+  // path is sequence of `.word` , `[number]`, `["string"]`
+  getJsonElement (blob: any, path: string, origPath = path): string|null {
```

Comment:
No f** way this belongs to the ContractInteractor! Please move JSON magic related code to a 'gas price oracle' class.